pkg/kf/commands/apps/apps.go

```diff
@@ -37,47 +40,100 @@ func NewAppsCommand(p *config.KfParams, appsClient apps.Client) *cobra.Command {
 			}
 			cmd.SilenceUsage = true
 
-			fmt.Fprintf(cmd.OutOrStdout(), "Getting apps in namespace: %s\n", p.Namespace)
+			fmt.Fprintf(cmd.OutOrStdout(), "Getting apps in space %s\n", p.Namespace)
 
 			apps, err := appsClient.List(p.Namespace)
 			if err != nil {
 				return err
 			}
-			fmt.Fprintf(cmd.OutOrStdout(), "Found %d apps in namespace %s\n", len(apps), p.Namespace)
 			fmt.Fprintln(cmd.OutOrStdout())
 
-			// Emulating:
-			// https://github.com/knative/serving/blob/master/config/300-service.yaml
 			w := tabwriter.NewWriter(cmd.OutOrStdout(), 8, 4, 1, ' ', tabwriter.StripEscape)
-			fmt.Fprintln(w, "NAME\tDOMAIN\tLATEST CREATED\tLATEST READY\tREADY\tREASON")
+			fmt.Fprintln(w, "name\t\trequested state\t\tinstances\tmemory\tdisk\turls")
 			for _, app := range apps {
-				var status, reason string
-				if cond := app.Status.GetCondition("Ready"); cond != nil {
-					status = fmt.Sprintf("%v", cond.Status)
-					reason = cond.Reason
+
+				// Requested State
+				requestedState := "started"
+				if app.Spec.Instances.Stopped {
+					requestedState = "stopped"
+				} else if cond := app.Status.GetCondition("Ready"); cond != nil && cond.Status == "Pending" {
+					requestedState = "starting"
+				} else if !app.DeletionTimestamp.IsZero() {
+					requestedState = "deleting"
 				}
 
-				if !app.DeletionTimestamp.IsZero() {
-					reason = "Deleting"
+				// Instances
+				var instances string
+				switch {
+				case app.Spec.Instances.Exactly != nil:
+					instances = strconv.FormatInt(int64(*app.Spec.Instances.Exactly), 10)
+				case app.Spec.Instances.Min == nil && app.Spec.Instances.Max == nil:
+					instances = "?"
+				case app.Spec.Instances.Min != nil && app.Spec.Instances.Max != nil:
+					instances = fmt.Sprintf(
+						"%d - %d",
+						*app.Spec.Instances.Min,
+						*app.Spec.Instances.Max,
+					)
+				case app.Spec.Instances.Max != nil:
+					instances = fmt.Sprintf(
+						"0 - %d",
+						*app.Spec.Instances.Max,
+					)
+				case app.Spec.Instances.Min != nil:
+					instances = fmt.Sprintf(
+						"%d - ∞",
+						*app.Spec.Instances.Min,
+					)
 				}
 
-				if app.Name == "" {
-					continue
+				// Memory
+				var memory string
+				if containers := app.Spec.Template.Spec.Containers; len(containers) > 0 {
+					if mem, ok := containers[0].Resources.Requests["memory"]; ok {
+						memory = mem.String()
+					}
+				}
+
+				// Disk
+				// TODO(#431): Persistent disks
+				var disk string
+				if containers := app.Spec.Template.Spec.Containers; len(containers) > 0 {
+					if d, ok := containers[0].Resources.Requests["ephemeral-storage"]; ok {
```

Comment:
I think there are constants for both ephemeral-storage and memory, Julia's open PR should have them